packages/account-export/src/lib/helpers/importAccount.ts

```diff
@@ -0,0 +1,37 @@
+import CryptoJS from "crypto-js";
+
+import type { AccountImportData } from "@near-wallet-selector/core";
+
+export const encodeAccountData = (
```

Comment:
yes we do. This is prerequisite step to convert account data to string + web friendly format. 

Do you see a reason why we should remove this?

Reply:
It was my understanding that the encrypted data would be base64-encoded and not require any URL-escaping.

> Do you see a reason why we should remove this?

I mentioned it because it is only invoked in a test. It's not clear to me why we need it if we are passing encrypted data in the URL.

Reply:
Thanks for the correction. I misunderstood the use case of encode/decode account data. Will remove them on next commit